libtransmission/peer-mgr-wishlist.h

```diff
@@ -70,6 +71,8 @@ class Wishlist
             libtransmission::SimpleObservable<tr_torrent*, tr_peer*, tr_block_span_t>::Observer observer) = 0;
         [[nodiscard]] virtual libtransmission::ObserverTag observe_sequential_download_changed(
             libtransmission::SimpleObservable<tr_torrent*, bool>::Observer observer) = 0;
+        [[nodiscard]] virtual libtransmission::ObserverTag observe_sequential_download_from_piece_changed(
+            libtransmission::SimpleObservable<tr_torrent*, bool>::Observer observer) = 0;
```

Comment:
Right now this doesn't matter in practice, but still:

```suggestion
            libtransmission::SimpleObservable<tr_torrent*, tr_piece_index_t>::Observer observer) = 0;
```